apps/vr-tests-web-components/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.base.wc.json",
+  "compilerOptions": {
+    "target": "ES2019",
+    "module": "ESNext",
+    "noEmit": true,
+    "experimentalDecorators": true,
+    "resolveJsonModule": true,
+    "allowJs": true,
+    "jsx": "react",
+    "moduleResolution": "Node16",
+    "typeRoots": ["../../node_modules/@types"]
```

Comment:
this is redundant as it's specified within base config

```suggestion
```

Reply:
@miroslavstastny to comment

Reply:
Added by @TristanWatanabe in https://github.com/microsoft/fluentui/pull/28222. Perhaps copy&paste from existing FUIR9 screenshot tests.